var/spack/repos/builtin/packages/neurodamus-base/package.py

```diff
@@ -34,6 +34,7 @@ class NeurodamusBase(Package):
     url      = "ssh://bbpcode.epfl.ch/sim/neurodamus/bbp"
 
     version('master',      git=url, branch='master')
+    version('mousify',     git=url, branch='sandbox/leite/mousify')
```

Comment:
only this version is concern but I will merge this for now as we need this on BB5